static/src/javascripts/projects/common/modules/ui/message.spec.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import $ from 'lib/$';
+import _userPrefs from 'common/modules/user-prefs';
+import { Message } from './message';
+
+jest.mock('common/modules/user-prefs', () => ({
+    get: jest.fn(),
+    set: jest.fn(),
+}));
+
+jest.mock('common/modules/analytics/register', () => ({
+    begin: jest.fn(),
+}));
+
+const userPrefs: any = _userPrefs;
```

Comment:
Perhaps the type of this could be [`JestMockFn<*, *>`](https://github.com/guardian/frontend/blob/master/ui/src/__flow__/flow-typed/npm/jest_v20.x.x.js#L4)